src/Breadcrumbs/Breadcrumb.php

```diff
@@ -97,15 +92,15 @@ public function __construct($level, $type, $category, $message = null, array $me
     /**
      * Creates a new instance of this class configured with the given params.
      *
-     * @param string      $level    The error level of the breadcrumb
+     * @param Severity    $level    The error level of the breadcrumb
      * @param string      $type     The type of the breadcrumb
      * @param string      $category The category of the breadcrumb
      * @param string|null $message  Optional text message
      * @param array       $metadata Additional information about the breadcrumb
      *
      * @return static
      */
-    public static function create($level, $type, $category, $message = null, array $metadata = [])
+    public static function create(Severity $level, $type, $category, $message = null, array $metadata = [])
```

Comment:
Is this static method still useful, since it's the same as the constructor?

Reply:
When I created it its purpose was to let users have a way for example to use it as callback of functions that accept a `callable` (e.g. `array_map`). Maybe this use case is not really useful, so I will remove it